Library/ViewModels/ViewMoreRepliesCellViewModel.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+import KsApi
+import Prelude
+import ReactiveExtensions
+import ReactiveSwift
+
+public protocol ViewMoreRepliesCellViewModelInputs {
+  func configureWith()
+}
+
+public protocol ViewMoreRepliesCellViewModelOutputs {}
+
+public protocol ViewMoreRepliesCellViewModelType {
+  var inputs: ViewMoreRepliesCellViewModelInputs { get }
+  var outputs: ViewMoreRepliesCellViewModelOutputs { get }
+}
+
+public final class ViewMoreRepliesCellViewModel: ViewMoreRepliesCellViewModelType,
+  ViewMoreRepliesCellViewModelInputs, ViewMoreRepliesCellViewModelOutputs {
+  public init() {}
+
+  fileprivate let configureWithProperty = MutableProperty(())
+  public func configureWith() {
```

Comment:
Not doing anything with this here but added it in to the body of the `configureWith` method the `ValueCell` protocol requires for conformance.

Reply:
Actually I don't think we need this entire file. It's never used.

Reply:
Deleting this too.